### ISSUE TITLE
Proxy interior object for TARDISes with no interior

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#1028]
In many extensions, CustomHooks do not have the `if not IsValid(int) then return end` line and rely on interior existing. 

![image](https://github.com/MattJeanes/TARDIS/assets/32988384/8778a561-3593-4cee-8c5f-5ecd236fe378)

Since fixing all those extensions would take a while, perhaps we should just add any object with all interior's parameter names and empty functions?

e.g.
```lua
function ENT:CallHook(...)
...
if not IsValid(int) then
    local int = {}
    -- do this for all interior existing functions, probably via metatable fields:
    
    int.ApplyTextureSet = function() end
    
end

...

CustomHooks.x.func(ext, int, ...)

...
```

**Comment by AmyJeanes** [comment]
I disagree with doing this, a fake interior proxy will just make things even more confusing. I know some are no longer maintained but this is not something I think we should mess with.
